src/TLuaInterpreter.cpp

```diff
@@ -5621,23 +5644,45 @@ int TLuaInterpreter::sendTelnetChannel102(lua_State* L)
 {
     string msg;
     if (!lua_isstring(L, 1)) {
-        lua_pushstring(L, "sendTelnetChannel102: wrong argument type");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "sendTelnetChannel102: bad argument #1 type (message bytes {2 characters} as string expected, got %s!)",
+                        luaL_typename(L, 1));
+        return lua_error(L);
     } else {
         msg = lua_tostring(L, 1);
+        if (msg.length() != 2) {
+            lua_pushnil(L);
+            lua_pushfstring(L, "invalid message of length %d supplied, it should be two bytes (may use lua \\### for each byte where ### is a number between 1 and 254)",
```

Comment:
`\###` such as `\200` (for the byte with unsigned integer value of 200)  is how you specify a byte value outside of the range of ASCII characters - but to show the `\` in a message it has to be escaped as `\\` - basically it is saying how to provide those byte values in a lua string. E.g in the section "Unicode_Lua_programs" in the "[Lua Unicode](http://lua-users.org/wiki/LuaUnicode)" page of the Lua Wiki it mentions:
> Literal Unicode strings can appear in your lua programs. Either a UTF-8 encoded string can appear directly with 8-bit characters or you can use the \ddd syntax (note that ddd is a decimal number, unlike some other languages). However, there is no facility for encoding multi-octet sequences (such as \U+20B4); you would need to either manually encode them to UTF-8, or insert individual octets in the correct big-endian/little-endian order (for UTF-16 or UTF-32).

I mention this escape form because I also use it to report the breakdown of non-ASCII characters (i.e. those that will use either multiple bytes) and non-printable characters (ASCII or not) into this form in the Text/Byte analyser in PR https://github.com/Mudlet/Mudlet/pull/1579 (so that a user can know how to enter a non-ASCII codepoint/grapheme into a Lua script/string)...

Reply:
Okay!